ToggleButton/ToggleButton.html

```diff
@@ -1,4 +1,4 @@
 <template aria-pressed="{{checked}}" attach-point="focusNode" on-click="{{toggle}}">
-	<span attach-point="iconNode" class="d-reset d-inline d-icon {{this.checked && this.checkedIconClass ? this.checkedIconClass : (this.iconClass ? this.iconClass : 'd-hidden')}}"
-	></span><span attach-point="labelNode" class="d-reset d-inline" d-hidden="{{!this.checkedLabel && !this.label}}">{{this.checked && this.checkedLabel ? this.checkedLabel : this.label}}</span>
+	<span attach-point="iconNode" class="d-reset d-inline d-icon {{this.checked &amp;&amp; this.checkedIconClass ? this.checkedIconClass : (this.iconClass ? this.iconClass : 'd-hidden')}}"
+	></span><span attach-point="labelNode" class="d-reset d-inline" d-hidden="{{!this.checkedLabel &amp;&amp; !this.label}}">{{this.checked &amp;&amp; this.checkedLabel ? this.checkedLabel : this.label}}</span>
```

Comment:
There might be a reason, but I wonder why you escape the `&&` into `&amp;&amp;`. 
https://github.com/ibm-js/delite/blob/master/docs/handlebars.md doesn't mention such a need, and at least in some browsers `&&` seems to work just fine.